### ISSUE TITLE
oletools 0.47 - issues with mraptor and olevba

### ISSUE THREAD

**Issue by decalage2** [decalage2/oletools#51]
oletools 0.47 has some issues:

1 - mraptor.py (0.04) gives me errors for tablestream and colorglass...so
I over-copied both folders with the folders from version 0.46 and it's
working.
=> moved to issue #57 

2 - olevba.py --decode option is not extracting the content of an
UserForm. VBA FORM STRING is missing.
=> now moved to issue #60.

3 - when running mraptor: 
```text
local variable 'modulename_unicode_modulename_unicode'  referenced before assignment - triggered in olevba
```

4 - olevba: 
```text
ERROR    Unhandled exception in main: 'ProcessingError' object has no attribute 'orig_exception'
Traceback (most recent call last):
  File "/usr/local/bin/olevba", line 3292, in main
    print '%-12s %s - %s' % ('!ERROR', filename, exc.orig_exception)
AttributeError: 'ProcessingError' object has no attribute 'orig_exception'
```

**Comment by decalage2** [comment]
fixed 3 and 4 in commit 9225bf167cfbc821eb27624e37e6d3fb80ef7a81.

**Comment by decalage2** [comment]
For the colorclass issue on MacOSX, need to test the solution suggested in https://github.com/Robpol86/colorclass/pull/20
